frontend/public/components/factory/list-page.jsx

```diff
@@ -434,7 +434,7 @@ export const ListPage = withFallback((props) => {
       showTitle={showTitle}
       textFilter={textFilter}
       title={title}
-      badge={badge || getBadgeFromType(ko.badge)}
```

Comment:
This is not the best way to go with this solution... Please re-read the ticket @sahil143 - I posted a PR link to what Sam did to solve this issue... we should look to mirror that here.

Reply:
Checked on the viability of that solution, and it's not going to work for this use-case. We still cannot do this change, it breaks the usage of badges throughout the list pages.

Reply:
Turns out there is a spot in the code that saves this chunk of code from needing to to this logic... and that's in the [resource-list.tsx](https://github.com/openshift/console/blob/master/frontend/public/components/resource-list.tsx#L56) file